src/views/LearningAndHobbies/index.jsx

```
@@ -0,0 +1,247 @@
+            <div className="flex justify-center text-gray-400">
+              No Videos Found
+            </div>
+          ) : null}
+        </div>
+      ) : (
+        <div className="mt-10">
+          {isVideoFetching ? (
+            <Spinner />
+          ) : videos?.length > 0 ? (
+            videos?.map((video) => (
+              <div className="mb-5">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)